diff --git a/node_modules/@connect/app/node_modules/react-native/React/Base/RCTModuleData.mm b/node_modules/@connect/app/node_modules/react-native/React/Base/RCTModuleData.mm
index ba2dbd5..aee870a 100644
--- a/node_modules/@connect/app/node_modules/react-native/React/Base/RCTModuleData.mm
+++ b/node_modules/@connect/app/node_modules/react-native/React/Base/RCTModuleData.mm
@@ -330,9 +330,13 @@ - (void)gatherConstants
 
 - (dispatch_queue_t)methodQueue
 {
-  (void)[self instance];
-  RCTAssert(_methodQueue != nullptr, @"Module %@ has no methodQueue (instance: %@, bridge.valid: %d)",
-            self, _instance, _bridge.valid);
+  // Changes from: https://github.com/facebook/react-native/commit/dc893756b82051d48d6a776aeef1186c31cf310d
+
+  if (_bridge.valid) {
+    id instance = self.instance;
+    RCTAssert(_methodQueue != nullptr, @"Module %@ has no methodQueue (instance: %@)",
+              self, instance);
+  }
   return _methodQueue;
 }
 
diff --git a/node_modules/@connect/app/node_modules/react-native/React/CxxModule/RCTNativeModule.mm b/node_modules/@connect/app/node_modules/react-native/React/CxxModule/RCTNativeModule.mm
index 5d4a2b8..99bae0e 100644
--- a/node_modules/@connect/app/node_modules/react-native/React/CxxModule/RCTNativeModule.mm
+++ b/node_modules/@connect/app/node_modules/react-native/React/CxxModule/RCTNativeModule.mm
@@ -71,17 +71,21 @@
     invokeInner(weakBridge, weakModuleData, methodId, std::move(params));
   };
 
-  if (m_bridge.valid) {
-    dispatch_queue_t queue = m_moduleData.methodQueue;
-    if (queue == RCTJSThread) {
-      block();
-    } else if (queue) {
-      dispatch_async(queue, block);
-    }
-  } else {
-    RCTLogWarn(@"Attempted to invoke `%u` (method ID) on `%@` (NativeModule name) with an invalid bridge.",
-               methodId, m_moduleData.name);
+  // Changes from: https://github.com/facebook/react-native/commit/dc893756b82051d48d6a776aeef1186c31cf310d
+
+  dispatch_queue_t queue = m_moduleData.methodQueue;
+  if (queue == RCTJSThread) {
+    block();
+  } else if (queue) {
+    dispatch_async(queue, block);
   }
+  
+#ifdef RCT_DEV
+  if (!queue) {
+    RCTLog(@"Attempted to invoke `%u` (method ID) on `%@` (NativeModule name) without a method queue.",
+           methodId, m_moduleData.name);
+  }
+#endif
 }
 
 MethodCallResult RCTNativeModule::callSerializableNativeHook(unsigned int reactMethodId, folly::dynamic &&params) {
diff --git a/node_modules/@connect/app/node_modules/react-native/React/Views/ScrollView/RCTScrollView.m b/node_modules/@connect/app/node_modules/react-native/React/Views/ScrollView/RCTScrollView.m
index 58bff0f..cee8b70 100644
--- a/node_modules/@connect/app/node_modules/react-native/React/Views/ScrollView/RCTScrollView.m
+++ b/node_modules/@connect/app/node_modules/react-native/React/Views/ScrollView/RCTScrollView.m
@@ -29,6 +29,7 @@ - (instancetype)initWithEventName:(NSString *)eventName
                          reactTag:(NSNumber *)reactTag
           scrollViewContentOffset:(CGPoint)scrollViewContentOffset
            scrollViewContentInset:(UIEdgeInsets)scrollViewContentInset
+   scrollViewAdjustedContentInset:(UIEdgeInsets)scrollViewAdjustedContentInset
             scrollViewContentSize:(CGSize)scrollViewContentSize
                   scrollViewFrame:(CGRect)scrollViewFrame
               scrollViewZoomScale:(CGFloat)scrollViewZoomScale
@@ -41,6 +42,7 @@ @implementation RCTScrollEvent
 {
   CGPoint _scrollViewContentOffset;
   UIEdgeInsets _scrollViewContentInset;
+  UIEdgeInsets _scrollViewAdjustedContentInset;
   CGSize _scrollViewContentSize;
   CGRect _scrollViewFrame;
   CGFloat _scrollViewZoomScale;
@@ -55,6 +57,7 @@ - (instancetype)initWithEventName:(NSString *)eventName
                          reactTag:(NSNumber *)reactTag
           scrollViewContentOffset:(CGPoint)scrollViewContentOffset
            scrollViewContentInset:(UIEdgeInsets)scrollViewContentInset
+   scrollViewAdjustedContentInset:(UIEdgeInsets)scrollViewAdjustedContentInset
             scrollViewContentSize:(CGSize)scrollViewContentSize
                   scrollViewFrame:(CGRect)scrollViewFrame
               scrollViewZoomScale:(CGFloat)scrollViewZoomScale
@@ -68,6 +71,7 @@ - (instancetype)initWithEventName:(NSString *)eventName
     _viewTag = reactTag;
     _scrollViewContentOffset = scrollViewContentOffset;
     _scrollViewContentInset = scrollViewContentInset;
+    _scrollViewAdjustedContentInset = scrollViewAdjustedContentInset;
     _scrollViewContentSize = scrollViewContentSize;
     _scrollViewFrame = scrollViewFrame;
     _scrollViewZoomScale = scrollViewZoomScale;
@@ -97,6 +101,12 @@ - (NSDictionary *)body
       @"bottom": @(_scrollViewContentInset.bottom),
       @"right": @(_scrollViewContentInset.right)
     },
+    @"adjustedContentInset": @{
+      @"top": @(_scrollViewAdjustedContentInset.top),
+      @"left": @(_scrollViewAdjustedContentInset.left),
+      @"bottom": @(_scrollViewAdjustedContentInset.bottom),
+      @"right": @(_scrollViewAdjustedContentInset.right)
+    },
     @"contentSize": @{
       @"width": @(_scrollViewContentSize.width),
       @"height": @(_scrollViewContentSize.height)
@@ -1135,10 +1145,17 @@ - (void)sendScrollEventWithName:(NSString *)eventName
     _coalescingKey++;
     _lastEmittedEventName = [eventName copy];
   }
+  UIEdgeInsets adjustedContentInset;
+  if (@available(iOS 11.0, *)) {
+    adjustedContentInset = scrollView.adjustedContentInset;
+  } else {
+    adjustedContentInset = scrollView.contentInset;
+  }
   RCTScrollEvent *scrollEvent = [[RCTScrollEvent alloc] initWithEventName:eventName
                                                                  reactTag:self.reactTag
                                                   scrollViewContentOffset:scrollView.contentOffset
                                                    scrollViewContentInset:scrollView.contentInset
+                                           scrollViewAdjustedContentInset:adjustedContentInset
                                                     scrollViewContentSize:scrollView.contentSize
                                                           scrollViewFrame:scrollView.frame
                                                       scrollViewZoomScale:scrollView.zoomScale
@@ -1159,6 +1176,7 @@ - (void)sendFakeScrollEvent:(NSNumber *)reactTag
                                                                      reactTag:reactTag
                                                       scrollViewContentOffset:CGPointZero
                                                        scrollViewContentInset:UIEdgeInsetsZero
+                                               scrollViewAdjustedContentInset:UIEdgeInsetsZero
                                                         scrollViewContentSize:CGSizeZero
                                                               scrollViewFrame:CGRectZero
                                                           scrollViewZoomScale:0
